src/browser-frontend/reducers/pages-reducers.js

```diff
@@ -43,20 +43,21 @@ function addPage(state, { payload: { url, parentId, background } = {} }) {
 
 function removePage(state, { payload: { pageId, withoutSelectingNextLogicalPage } }) {
   return state.withMutations((mut) => {
-    const pageIndex = state.ui.pages.displayOrder.findIndex(id => id === pageId);
-    const pageCount = state.ui.pages.displayOrder.count();
     const selectedId = state.ui.pages.selectedId;
+    const idToRemove = pageId || selectedId;
```

Comment:
Benefit: less mental overhead remembering what each action does, less code in reducers. In this particular case it's not that big of a difference though.